README.md

```diff
@@ -97,31 +78,32 @@ $ http://DOCKERHOST:9200/_cluster/health?pretty
 }
 ```
 #### To build the plugin binary:
-Get the source by running a `go get` to fetch the code:
+
+Fork https://github.com/intelsdi-x/snap-plugin-collector-elasticsearch
```

Comment:
Could you add new line after this sentence it will look better? Now this line is is joined with the next line.